kvmagent/kvmagent/test/test_usb.py

```
@@ -0,0 +1,38 @@
+import unittest
+
+import pytest
+
+from zstacklib.utils import linux
+
+
+class Test(unittest.TestCase):
+    def test_allocate_usb_controller(self):
+        usb_manager = linux.VmUsbManager()
+        # invalid usb version
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)